github/github.go

```diff
@@ -603,6 +612,8 @@ func CheckResponse(r *http.Response) error {
 			abuseRateLimitError.RetryAfter = &retryAfter
 		}
 		return abuseRateLimitError
+	case r.StatusCode == http.StatusAccepted:
+		return &AcceptedError{}
```

Comment:
I'm tempted to suggest we turn this into a value rather than a pointer. It's an empty struct. Pointer seems unneeded and unhelpful.

```Go
return AcceptedError{}
```